lib/matplotlib/axes/_base.py

```diff
@@ -559,13 +559,21 @@ def __init__(self, fig, rect,
             bottom=rcParams['xtick.bottom'] and rcParams['xtick.minor.bottom'],
             left=rcParams['ytick.left'] and rcParams['ytick.minor.left'],
             right=rcParams['ytick.right'] and rcParams['ytick.minor.right'],
+            labelleft=rcParams['ytick.labelleft'] and
+            rcParams['ytick.minor.left'],
+            labelright=rcParams['ytick.labelright'] and
+            rcParams['ytick.minor.right'],
             which='minor')
 
         self.tick_params(
             top=rcParams['xtick.top'] and rcParams['xtick.major.top'],
             bottom=rcParams['xtick.bottom'] and rcParams['xtick.major.bottom'],
             left=rcParams['ytick.left'] and rcParams['ytick.major.left'],
             right=rcParams['ytick.right'] and rcParams['ytick.major.right'],
+            labelleft=rcParams['ytick.labelleft'] and
+            rcParams['ytick.major.left'],
```

Comment:
You should indent a 8 spaces or so here...

Reply:
yep sounds good.

Reply:
Actually when I indented like the below I got some PEP 8 complains. Was this what you meant? 

Alternatively I can have this in a single line but it breaches 80 line margin.

```
        labelleft=rcParams['ytick.labelleft'] and
            rcParams['ytick.major.left'],
```

Reply:
Put the RHS in parentheses and then it should work with indenting...